src/utils/components/PreviewFeatureAlert/PreviewFeatureAlert.tsx

```diff
@@ -28,7 +31,7 @@ const PreviewFeatureAlert: FC<PreviewFeatureAlertProps> = ({ onClose }) => {
         </StackItem>
         <StackItem>
           <span onClick={onClose}>
-            <Link to={OVERVIEW_SETTINGS_TAB_PATH}>
+            <Link to={getOverviewSettingsPath(activeNamespace)}>
```

Comment:
working also with all projects?

Reply:
This link is only accessible via the Network interface list page in VM details. Is it possible to have a VM in the All Projects namespace? I updated it to work with All projects just in case, but it shouldn't matter.

Reply:
well if its vm details page so it must have a ns different from all.
But this improvements I no harm